### ISSUE TITLE
Problem with loading any movie's page

### ISSUE THREAD

**Issue by AlexS-GitHub** [OstlerDev/PopcornTV#167]
On my Apple TV, when a movie is selected, it hangs at "Accessing Trailers," for a while, then reports that Trailers (PopcornTV) is unavailable.

     Thanks!
          -Alex

**Comment by AlexS-GitHub** [comment]
I have read OstlerDev's comment on issue 166. I will try to reinstall the certificate, and report back.

**Comment by AlexS-GitHub** [comment]
Perhaps it is a problem on my end? Reinstalling the certificate did not work, I am almost positive it is a problem on my end...

Here is my log.

http://pastebin.com/SdiwCveX

**Comment by OstlerDev** [comment]
Reinstalling the certificate usually only works when accessing trailers for the first time throws an error. If you're able to get in at all that means the certificate is working. Are you on the latest master branch?

**Comment by AlexS-GitHub** [comment]
Yes, I am on the latest master branch. Still the same problem. Here's the log from this run.

http://pastebin.com/ymY7x5eT

**Comment by OstlerDev** [comment]
Very interesting, it looks like there is an issue where the binding for that version of Linux does not exist for some reason. I will open up a bug report on their wiki, but here is what you should do for now.

1. Go into the PopcornTV Directory
2. Go into ``node_modules/node-images/bindings/linux/ia32``
3. Duplicate the folder ``0.9.11`` and rename it ``0.12.0``
4. Restart PopcornTV

If that does not work for you, then you will need to go into the settings and disable Fanart until that bug gets fixed.

**Comment by AlexS-GitHub** [comment]
Hey, I just had a night watching movies. It turns out that the renaming was ineffective. However, disabling Fanart was effective. Thanks. I will leave this issue open for those of you who wish to comment further.

**Comment by OstlerDev** [comment]
What version of Linux are you running and what are the spec's of your machine that runs them?

**Comment by OstlerDev** [comment]
@AlexS-GitHub What version of Linux are you running and what are the spec's of your machine that runs them?

**Comment by AlexS-GitHub** [comment]
I'm sorry, @OstlerDev, I was on vacation. My machine is running a variation of Debian Wheezy. It uses 20 GB (I don't need much, it is a lightweight server), and 512 MB of RAM.

**Comment by OstlerDev** [comment]
@AlexS-GitHub I am going to need you to compile a binding for the images, you can find a tutorial here: https://github.com/PopcornTV/node-images/wiki/How-to-build-a-binding

Once you have created the binding let me know and I can add it to the repository.

**Comment by AlexS-GitHub** [comment]
Alright, I can probably get the compiled binding to you by 10:00 PM Eastern time today.

I will send the final product via pull request, what should the binding folder be named?

**Comment by OstlerDev** [comment]
It will be named binding.node and should be in the build/releases folder. In the pull request you should put it in /bindings/linux/ia32/0.12.0

**Comment by AlexS-GitHub** [comment]
The sourceforge servers appear to be on and off, wget in the third-party script is currently retrying again and again to download files from cznic.fl.sourceforge.net. Here is where the script is so far. It's been running for about 15 minutes so far.

http://pastebin.com/q71ZmswE

I will leave it on until about midnight Eastern, and hopefully the script has done its job by then.

**Comment by OstlerDev** [comment]
I believe some of the links are dead, I will need to update them.

**Comment by AlexS-GitHub** [comment]
Ok, wow, that was fast.

**Comment by AlexS-GitHub** [comment]
I am done for the day, I will give you time to update links, and tomorrow I can try. Thanks.

**Comment by OstlerDev** [comment]
@AlexS-GitHub I updated the install script a bit ago, let me know when you can get the binding.

**Comment by AlexS-GitHub** [comment]
OK, I saw the gyp repository. I will attempt the build again.

**Comment by AlexS-GitHub** [comment]
Oh, brainfart. The gyp has nothing to do with the script =D

**Comment by AlexS-GitHub** [comment]
The last command, ironically, failed. "nw-gyp rebuild"

Here is the log:

http://pastebin.com/jCf9NET4

**Comment by OstlerDev** [comment]
Ah, I need to modify the install script, that is my bad. Use ``nw-gyp -v v0.12.3``

You can see that 0.12.3 is the latest version here: https://github.com/nwjs/nw.js/releases

**Comment by AlexS-GitHub** [comment]
Yes. Another problem was that my Debian didn't have cmake, and I thought that was standard. 

So I gave it another run, but I saw the exact same error as before.
